src/components/ControlInput.vue

```diff
@@ -583,7 +586,7 @@ export default {
 
             this.lastTypingTime = Date.now();
         },
-        stopTyping() {
+        stopTyping(messageSent) {
```

Comment:
`messageSent` would be better made more generic, something like `sendStopPause`. There may be a case where we also don't want to send stop/pause that isn't related to sending a message.